sqekin/main.js

```diff
@@ -0,0 +1,67 @@
+const STATUS_TODO = "To Do";
+const STATUS_INWORK = "In Work";
+const STATUS_DONE = "Done";
+
+const taskList = {
+  list: {},
+};
+
+// Функция для добавления задачи
+const addTask = (taskName) => (taskList.list[taskName] = STATUS_TODO);
+
+// Функция для изменения статуса задачи
+const changeStatus = (taskName, status) => (taskList.list[taskName] = status);
+
+// Функция удаления задачи
+const deleteTask = (taskName) => delete taskList.list[taskName];
+
+// Функция вывода результата
+function showList() {
+  let counter = 0;
+  let todoTasks = "";
+  let inworkTasks = "";
+  let doneTasks = "";
+
+  for (const task in taskList.list) {
+    switch (taskList.list[task]) {
+      case STATUS_TODO:
+        todoTasks += `    ${task}\n`;
```

Comment:
Можно еще добавить, чтоб совсем по красоте было)
```suggestion
        todoTasks += `    "${task}"\n`;
```